Code/Framework/AzToolsFramework/AzToolsFramework/Viewport/ViewportMessages.h

```diff
@@ -347,20 +341,39 @@ namespace AzToolsFramework
 
     //! Wrap EBus call to retrieve manipulator line bound width.
     //! @note It is possible to pass AzFramework::InvalidViewportId to perform a Broadcast as opposed to a targeted Event.
-    inline float ManipulatorLineBoundWidth(AzFramework::ViewportId viewportId)
+    inline float ManipulatorLineBoundWidth(const AzFramework::ViewportId viewportId)
```

Comment:
[minor] it seems unnecessarily restrictive for this functions' implementation to make `viewportId` parameter const since it's passed by value.

Reply:
it's marked `const` only in the implementation (which is the same as the interface here as the function is `inline`) to indicate that at no point in the function `viewportId` ever changes - I think this is useful information to the reader so I'd like to keep it (I do this often where in the interface the type is pass by value and not marked const, but is annotated `const` in the implementation to show that variable is never modified or assigned to)